k8s/jiva/snapshot-cleanup.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+usage()
+{
+    echo "Usage: bash script.sh <controller_pod_name> <application_namespace> <numbber_of_snapshots_to_delete>"
+    echo "WARNING: This script need to be run from client that have access to the cluster. This script can run for longer time based on the amount of data in the snapshots that need to be deleted. Please don't cancel the operation while it is running. If the script gets broken due to any network issue, you need to run unblock_service.sh script"
```

Comment:
This warning message needs to be printed - in a separate function and user should be prompted to to say yes to continue and no to exit.